src/wp-admin/contribute.php

```diff
@@ -0,0 +1,132 @@
+			<p><?php _e( 'WordPress may thrive on technical contributions, but you don&#8217;t have to code to contribute. Here are some of the ways you can make an impact without a single line of code:' ); ?></p>
+			<ul>
+				<li><?php _e( '<strong>Share</strong> your knowledge in the WordPress support forums' ); ?></li>
+				<li><?php _e( '<strong>Write</strong> or improve documentation for WordPress' ); ?></li>
+				<li><?php _e( '<strong>Translate</strong> WordPress into your local language' ); ?></li>
+				<li><?php _e( '<strong>Create</strong> and improve WordPress educational materials' ); ?></li>
+				<li><?php _e( '<strong>Promote</strong> the WordPress project to your community' ); ?></li>
+				<li><?php _e( '<strong>Take</strong> photos for or help curate the WordPress Photos Directory' ); ?></li>
+				<li><?php _e( '<strong>Organize</strong> or participate in local WordPress Meetups and WordCamps' ); ?></li>
+				<li><?php _e( '<strong>Lend</strong> your creative imagination to the WordPress UI designy' ); ?></li>
+				<li><?php _e( '<strong>Edit</strong> videos and add captions to WordPress.tv' ); ?></li>
+				<li><?php _e( '<strong>Explore</strong> ways to reduce the environmental impact of millions of websites around the world' ); ?></li>
+
+			</ul>
+		</div>
+		<div class="column">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='300' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+	</div>
+	<div class="about__section has-2-columns is-wider-right">
+		<div class="column">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='300' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+		<div class="column is-vertically-aligned-center">
+			<h3><?php _e( 'Code-based Contribution' ); ?></h3>
+			<p><?php _e( 'If you do code, or want to, you can contribute technically in numerous ways:' ); ?></p>
+			<ul>
+				<li><?php _e( '<strong>Find</strong> and report bugs in the WordPress core software' ); ?></li>
+				<li><?php _e( '<strong>Test</strong> new releases and proposed features for the block editor' ); ?></li>
+				<li><?php _e( '<strong>Write</strong> and submit patches to fix bugs or help build new features' ); ?></li>
+				<li><?php _e( '<strong>Contribute</strong> to the code, improve the UX, and test the WordPress app' ); ?></li>
+			</ul>
+			<p><?php _e( 'WordPress embraces new technologies, while being committed to backward compatibility. We use the following languages and libraries:' ); ?></p>
+			<ul>
+				<li><?php _e( 'For Core and Block Editor: HTML, CSS, PHP, SQL, JavaScript and React' ); ?></li>
+				<li><?php _e( 'For the WordPress app: Kotlin, Java, Swift, Objective-C, Vue, Python and TypeScript' ); ?></li>
+			</ul>
+		</div>
+	</div>
+
+	<div class="about__section">
```

Comment:
You could try using the style helper classes here, `is-feature` will make the text larger & add some padding, and `has-subtle-background-color` adds a light background color.

![Screen Shot 2023-07-11 at 10 43 33](https://github.com/WordPress/wordpress-develop/assets/541093/62adccf4-0458-4f67-b307-ba88e0f4d82c)

```suggestion
	<div class="about__section is-feature has-subtle-background-color">
```